openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/LightGBMUtils.java

```diff
@@ -91,13 +118,30 @@ static CpuArchitecture getCpuArchitecture(final String cpuArchName) {
      */
     static private void loadSharedLibraryFromJar(
             final String sharedLibResourceName,
-            final CpuArchitecture cpuArchitecture
+            final CpuArchitecture cpuArchitecture,
+            final boolean isAlpine
     ) throws IOException {
 
-        logger.debug("Loading LightGBM shared lib: {} for {}.", sharedLibResourceName, cpuArchitecture);
+        if (isAlpine && cpuArchitecture.getLgbmNativeLibsFolder().equals("arm64")) {
+            throw new IOException("Trying to use LightGBM on an unsupported architecture arm64 on Alpine.");
```

Comment:
Maybe -> Trying to use LightGBM on Alpine with unsupported arm64 architecture.

Reply:
Thanks, I changed the message in the latest commit.